src/main/java/net/masterthought/cucumber/ReportBuilder.java

```diff
@@ -64,45 +56,46 @@
      */
     private boolean wasTrendsFileSaved = false;
 
-    public ReportBuilder(List<String> jsonFiles, Configuration configuration) {
-        this.jsonFiles = jsonFiles;
+    public ReportBuilder(Configuration configuration) {
         this.configuration = configuration;
         reportParser = new ReportParser(configuration);
     }
 
+    @Deprecated
+    public ReportBuilder(List<String> jsonFiles, Configuration configuration) {
+        this(configuration);
+        this.jsonFiles = jsonFiles;
+    }
+
     /**
      * Parses provided files and generates the report. When generating process fails
      * report with information about error is provided.
      *
      * @return stats for the generated report
+     * @deprecated use {@link #generateReportsFromFiles(List)} instead
      */
+    @Deprecated
     public Reportable generateReports() {
-        Trends trends = null;
+        return generateReportsFromFiles(jsonFiles);
+    }
 
+    /**
+     * Parses provided files and generates the report. When generating process fails
+     * report with information about error is provided.
+     *
+     * @return stats for the generated report
+     */
+    public Reportable generateReportsFromFiles(List<String> pathsTojsonFiles) {
```

Comment:
this changes the API - will you update also maven and jenkins plugin which use this library as dependency?